Source/DSPSpy/tests/andc_ls_test.ds

```diff
@@ -14,7 +14,7 @@ include "dsp_base.inc"
 
 	lri $AC0.M, #0x0004
 	lri $AC1.M, #0x1234
-	andc'ls $AC1.M : $AX0.L, $AC0.M
+	cw 0x3d80 ; andc'ls $AC1.M : $AX0.L, $AC0.M
```

Comment:
What's going on here?

Reply:
`andc'ls` isn't a valid instruction, due to the way extension opcodes work when the first nybble is 3 (the main opcode is 9 bits, and the extension opcode is 7 bits).  That bit pattern actually encodes `lsrnr`.  I think this test is what originally found that 9-bit opcode behavior.

I ran into that when I tried to compile all of the tests to make sure they still worked after the `test_main` change.